recipe/build-shared.sh

```diff
@@ -13,6 +13,7 @@ then
     # to improve performance, disable checks intended for debugging
     CXXFLAGS="$CXXFLAGS -DNDEBUG"
 elif [ "$(uname)" == "Darwin" ];
+    export SED="${BUILD_PREFIX}/bin/sed"
 then
```

Comment:
this looks like a typo.  then after a statement?

Reply:
Moved into it's own block.